index.bs

```diff
@@ -96,6 +96,25 @@ device running the user agent, and be accessed over a transport such as
 Universal Serial Bus (USB), Bluetooth Low Energy (BLE) or Near Field
 Communications (NFC).
 
+## Specification Roadmap ## {#spec-roadmap}
+
+While many W3C specifications are directed primarily to audiences comprising user agent developers and web application developers (i.e., Web authors, herein also known as <em>[RP] developers</em>), the character of Web Authentication requires that this specification also be understood and properly used by those whose responsibilities include making decisions regarding:
```

Comment:
Nit: "also be understood" seems at odds with the repetition of "[RP] developers" below. Maybe say something like "... requires that this specification be understood and correctly used by multiple audiences, namely:"

Reply:
done.